source/features/open-notification-in-new-tab.js

```diff
@@ -0,0 +1,16 @@
+import select from 'select-dom';
+import {registerShortcut} from './improve-shortcut-help';
+
+export default function () {
+	registerShortcut('notifications', 'shift o', 'Open Notification in New Tab');
```

Comment:
`Open Notification in New Tab` => `Open selected notification in new tab`?

Reply:
This is definitely more clear, thanks for taking the time to review!